Makefile

```
@@ -466,7 +466,7 @@ amd64-almalinux: almalinux8 almalinux8-clang almalinux8-dbg almalinux9 almalinux
 amd64-centos: centos9 centos9-clang centos9-dbg centos10 centos10-clang centos10-dbg
 amd64-debian: debian12 debian12-clang debian12-dbg debian13 debian13-clang debian13-dbg
 amd64-fedora: fedora42 fedora42-clang fedora42-dbg fedora43 fedora43-clang fedora43-dbg fedora44 fedora44-clang fedora44-dbg
-amd64-opensuse: opensuse15 opensuse15-clang opensuse15-dbg opensuse16 opensuse16-clang opensuse16-dbg
+amd64-opensuse: opensuse16 opensuse16-clang opensuse16-dbg
```

checkmake (0.3.2)

[warning] Target "amd64-opensuse" should be declared PHONY.

(phonydeclared)